app/views/teaching_events/index.html.erb

```diff
@@ -11,12 +11,12 @@
 
       <dl>
         <div>
-          <dt>DfE Train to Teach events</dt>
+          <dt>Department for Education (DfE) Train to Teach events</dt>
           <dd>Watch presentations, hear from teachers, speak to advisers and meet local training providers.</dd>
```

Comment:
```suggestion
          <dd>Watch presentations, hear from teachers, speak to advisers and meet local training providers. Run by the Department for Education (DfE).</dd>
```

Expanding the acronym in the title looks a bit clunky, especially on mobile, and distracts from the actual event type - I wonder whether this is a better alternative? Would also need applying to the Online Q&As as well if so.